app/resources/api/v2/tube_from_tube_creation_resource.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # @todo This documentation does not yet include a detailed description of what this resource represents.
```

Comment:
Since this is a new one can you add a brief description of what its doing in place of these todos.

Reply:
That's fair.  I guess I was hoping someone more familiar with the model and how it's used could write a better description.  But I admit I did just copy-paste this from other endpoints I've been creating.  I could probably add example usage to the documentation at least, since that's not model specific.

Reply:
'Resource for the api endpoint specifically for the creation of a single child tube from a single parent tube.'
maybe?

Looks like a 1:1 transfer only? Can't see anything else particularly special about it.